lib/nexus/nexus_plot.cpp

```diff
@@ -314,7 +315,7 @@ ecl_sum_type* nex::ecl_summary(const std::string& ecl_case, const NexusPlot& plt
     bool time_in_days = true;
 
     ecl_sum_type * ecl_sum = ecl_sum_alloc_writer( ecl_case.c_str(),
-        fmt_output,
+        format_output,
```

Comment:
Extremely ugly that the `bool format_output` comes as part of the `ecl_sum_alloc_writer( )` call - that should have been in the `ecl_sum_fwrite( )` call; but that is for another PR.